<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalProfiles1</class>
 <widget class="QWidget" name="PersonalProfiles1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64,64,65)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235,225,187)</string>
   </property>
   <property name="text">
    <string>Personal Profiles</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255)</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>20</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>90</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255)</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QListWidget" name="editList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>541</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="activeProfile">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>591</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255)</string>
   </property>
   <property name="text">
    <string>Active Profile</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255)</string>
   </property>
   <property name="text">
    <string>Edit Profile (doubleclick)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="selectList">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>440</y>
     <width>541</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255)</string>
   </property>
   <property name="text">
    <string>Select Profile (doubleclick)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Currently Active Profile</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
